src/site/content/en/blog/intersectionobserver/index.md

```diff
@@ -0,0 +1,143 @@
+
+<figure>
+  {% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/igJyLJwnl7OUShqsWmMY.gif", alt="Threshold animation.", width="320", height="649" %}
+</figure>
+
+### Any other options?
+
+As of now, there's only one additional option to the ones listed above. [`rootMargin`](https://w3c.github.io/IntersectionObserver#dom-intersectionobserverinit-rootmargin) allows you to specify the margins for the root, effectively allowing you to either grow or shrink the area used for intersections. These margins are specified using a CSS-style string, á la "`10px 20px 30px 40px`", specifying top, right, bottom and left margin respectively. To summarize, the `IntersectionObserver` options struct offers the following options:
+
+```js
+new IntersectionObserver(entries => {/* … */}, {
+  // The root to use for intersection.
+  // If not provided, use the top-level document's viewport.
+  root: null,
+  // Same as margin, can be 1, 2, 3 or 4 components, possibly negative lengths.
+  // If an explicit root element is specified, components may be percentages of the
+  // root element size.  If no explicit root element is specified, using a
+  // percentage is an error.
+  rootMargin: "0px",
+  // Threshold(s) at which to trigger callback, specified as a ratio, or list of
+  // ratios, of (visible area / total area) of the observed element (hence all
+  // entries must be in the range [0, 1]).  Callback will be invoked when the
+  // visible ratio of the observed element crosses a threshold in the list.
+  threshold: [0],
+});
+```
+
+### `<iframe>` magic
+
+`IntersectionObserver`s were designed specifically with ads services and social network widgets in mind, which frequently use `<iframe>` elements and could benefit from knowing whether they are in view. If an `<iframe>` observes one of its elements, both scrolling the `<iframe>` as well as scrolling the window *containing the `<iframe>`* will trigger the callback at the appropriate times. For the latter case, however, `rootBounds` will be set to `null` to avoid leaking data across origins.
+
+## What is `IntersectionObserver` *Not* about?
+
+Something to keep in mind is that `IntersectionObserver` is intentionally neither pixel perfect nor low latency. Using them to implement endeavours like scroll-dependent animations are bound to fail, as the data will be&mdash strictly speaking&mdash;out of date by the time you'll get to use it. The [explainer](https://github.com/WICG/IntersectionObserver/blob/gh-pages/explainer.md) has more details about the original use cases for `IntersectionObserver`.
```

Comment:
Missed a semi-colon. Also when do we have to use `&mdash;` as opposed to `—` out of interest?

```suggestion
Something to keep in mind is that `IntersectionObserver` is intentionally neither pixel perfect nor low latency. Using them to implement endeavours like scroll-dependent animations are bound to fail, as the data will be&mdash;strictly speaking&mdash;out of date by the time you'll get to use it. The [explainer](https://github.com/WICG/IntersectionObserver/blob/gh-pages/explainer.md) has more details about the original use cases for `IntersectionObserver`.
```